Server-Monks/nutshell
# Cp messages one

# Description

Created file structure and started building code for messages.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

It should render the two messages that we have in the API with the edit and delete buttons, the buttons don't do anything yet.



# Checklist:
